ledger/compact_merkle_tree.py

```diff
@@ -261,10 +261,24 @@ def leafCount(self) -> int:
     def nodeCount(self) -> int:
         return self.hashStore.nodeCount
 
-    def verifyConsistency(self, expectedLeafCount = -1) -> bool:
-        if expectedLeafCount > 0 and expectedLeafCount != self.leafCount:
+    @staticmethod
+    def get_expected_node_count(leaf_count):
```

Comment:
We will move it when we need it at other places, i don't like idea of moving things to general utility files, maybe later on if we have several kinds of trees and some functions apply to some of those and not others, we will create a `tree_utils` or something like that